ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -847,27 +847,29 @@ - (void)unmountChildComponentView:(UIView<RCTComponentViewProtocol> *)childCompo
 
 - (void)replaceNavigationBarViewsWithSnapshotOfSubview:(RNSScreenStackHeaderSubview *)childComponentView
 {
-  UINavigationItem *navitem = _screenView.controller.navigationItem;
-  UIView *snapshot = [childComponentView snapshotViewAfterScreenUpdates:NO];
-
-  // This code should be kept in sync with analogous switch statement in
-  // `+ [RNSScreenStackHeaderConfig updateViewController: withConfig: animated:]` method.
-  switch (childComponentView.type) {
-    case RNSScreenStackHeaderSubviewTypeLeft:
-      navitem.leftBarButtonItem.customView = snapshot;
-      break;
-    case RNSScreenStackHeaderSubviewTypeCenter:
-    case RNSScreenStackHeaderSubviewTypeTitle:
-      navitem.titleView = snapshot;
-      break;
-    case RNSScreenStackHeaderSubviewTypeRight:
-      navitem.rightBarButtonItem.customView = snapshot;
-      break;
-    case RNSScreenStackHeaderSubviewTypeSearchBar:
-    case RNSScreenStackHeaderSubviewTypeBackButton:
-      break;
-    default:
-      RCTLogError(@"[RNScreens] Unhandled subview type: %ld", childComponentView.type);
+  if (childComponentView.window != nil) {
```

Comment:
Also added this check since we got many warning of snapshotting a view that is not in hierarchy when dismissing many screens with header items at once.